src/assets.ts

```diff
@@ -716,22 +730,29 @@ export async function generateAssets(server: OmniSharpServer, selectedIndex?: nu
         let workspaceInformation = await serverUtils.requestWorkspaceInformation(server);
         if (workspaceInformation.MsBuild && workspaceInformation.MsBuild.Projects.length > 0) {
             const generator = new AssetGenerator(workspaceInformation);
-            let doGenerateAssets = await shouldGenerateAssets(generator);
-            if (!doGenerateAssets) {
+            let doOverwriteAssets = await shouldOverwriteAssets(generator);
+            if (doOverwriteAssets === 'Cancel') {
                 return; // user cancelled
             }
 
+            let overwrite = doOverwriteAssets === 'Yes';
+
             const operations: AssetOperations = {
                 addLaunchJson: generator.hasExecutableProjects(),
```

Comment:
>addLaunchJson [](start = 16, length = 13)

Should this be setting `updateLaunchJson` also? And should `addLaunchJson` be dependent on `overwrite`?

Reply:
Outdated.